generators/repo/src/common/util.ts

```diff
@@ -47,6 +47,16 @@ export const ensureDirectoryPath = async (directoryPath: string) => {
     }
 }
 
+export const ensureFilePath = async (filePath: string) => {
```

Comment:
Consider renaming to `isFilePath`.  `ensureFilePath` sounds like if it wasn't a file path it would create it for me.